CHANGELOG

```diff
@@ -1,3 +1,7 @@
+v1.0.3:
+  date: 2017-01-23
+  changes:
+    - Fixed "open" to use appName
```

Comment:
Good stuff - I've run into this issue (open not working) - thanks for raising a PR to fix this 😄  Hope it gets merged soon!

I think your change does more, however: it also fixes `options.open.callback` which isn't respected in v1.0.2 as opn v4.0.0 does nothing with the callback function passed in the current code as [opn v4.0.x only takes 1 (optional) second parameter](https://github.com/sindresorhus/opn#options). Do you think that should be added to the change log?